features/main/input_output.feature

```diff
@@ -179,7 +179,11 @@ Feature: DTO input and output
       },
       "@type": "User",
       "@id": "/users/1",
-      "dummy": "/dummies/1"
+      "dummy": {
```

Comment:
This should probably be handled better?

Reply:
It'd be better to have `@id` as a string no?